.github/workflows/spotify-box.yml

```diff
@@ -10,7 +10,7 @@ jobs:
    runs-on: ubuntu-latest
    environment: prod
    steps:
-     - uses: actions/checkout@v2
+     - uses: actions/checkout@master
```

Comment:
```suggestion
     - uses: actions/checkout@v3
```

Reply:
master is not the default branch of actions/checkout, and it should use fixed version to avoid unexpected upgrade confliction.